src/qt/rpcconsole.cpp

```diff
@@ -523,6 +525,16 @@ void RPCConsole::setNumBlocks(int count, const QDateTime& blockDate)
     ui->lastBlockTime->setText(blockDate.toString());
 }
 
+void RPCConsole::setMempoolSize(int numberOfTxs, size_t dynUsage)
+{
+    ui->mempoolNumberTxs->setText(QString::number(numberOfTxs));
+
+    if (dynUsage < 1024*1024)
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0, 'f', 2) + " KB");
+    else
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0/1024.0, 'f', 2) + " MB");
```

Comment:
Technically, B in SI means bel. B as byte is defined by IEEE 1541 (which
also defined b to mean bit).